src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = { ...state };
+  const result = [];
+
+  for (const obj of actions) {
```

Comment:
maybe, better name (const action of  actions)

Reply:
hmm... agree